src/main/java/seedu/address/logic/commands/EditUserCommand.java

```diff
@@ -37,8 +40,9 @@
             + PREFIX_MAJOR + "MAJOR "
             + PREFIX_FROM + "FROM "
             + PREFIX_TO + "TO "
-            + PREFIX_CAP + "CAP "
+            + PREFIX_CAP + "CAP\n"
             + "Example: " + COMMAND_WORD + " "
+            + PREFIX_DP + "/Users/nhamquochung/Desktop/test.png "
```

Comment:
I can't think of a better way than hardcoding this path, but we should be careful here when others will be testing our app